StayGo.Domain/Models/Bar.cs

```diff
@@ -0,0 +1,13 @@
+namespace StayGo.Domain.Models
+{
+    public class Bar
+    {
+        private int Id { get; set; }
+        private string Name { get; set; }
+        private string Address { get; set; }
+        private string Description { get; set; }
+        private string Type { get; set; } 
```

Comment:
What is bar type ?

Reply:
According to the scheme. There is a field type of cuisine